### PR TITLE
Add RTS service

### PR DESCRIPTION
**Pull request by d-apurva**
This PR adds a capability to manage OTC's RTS service.

### CODE REVIEW

**Comment by niuzhenguo** [comment]
Please resolve conflicts and upadte CHANGELOG!

**Comment by Savasw** [comment]
@niuzhenguo We have resolved the conflicts and updated changelog

File: opentelekomcloud/data_source_opentelekomcloud_rts_stack_v1_test.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+// PASS
```
Comment (niuzhenguo) [inline opentelekomcloud/data_source_opentelekomcloud_rts_stack_v1_test.go]: Nits: No need to have such test result comment.

File: opentelekomcloud/import_opentelekomcloud_rts_stack_v1_test.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package opentelekomcloud
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+// PASS
```
Comment (niuzhenguo) [inline opentelekomcloud/import_opentelekomcloud_rts_stack_v1_test.go]: ditto.

File: opentelekomcloud/resource_opentelekomcloud_rts_stack_v1_test.go [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+						"opentelekomcloud_rts_stack_v1.stack_1", "timeout_mins", "50"),
+					resource.TestCheckResourceAttr(
+						"opentelekomcloud_rts_stack_v1.stack_1", "status", "UPDATE_COMPLETE"),
+				),
+			},
+		},
+	})
+}
+
+// PASS
```
Comment (niuzhenguo) [inline opentelekomcloud/resource_opentelekomcloud_rts_stack_v1_test.go]: ditto.

File: CHANGELOG.md [hunk 4]
```diff
@@ -15,6 +15,9 @@ FEATURES:
 * **New Resource:** `opentelekomcloud_vpc_peering_connection_accepter_v2` ([#87](https://github.com/terraform-providers/terraform-provider-opentelekomcloud/issues/87))
 * **New Data Source:** `opentelekomcloud_sfs_file_system_v2` ([#92](https://github.com/terraform-providers/terraform-provider-opentelekomcloud/issues/92))
 * **New Resource:** `opentelekomcloud_sfs_file_system_v2` ([#92](https://github.com/terraform-providers/terraform-provider-opentelekomcloud/issues/92))
+* **New Data Source:** `opentelekomcloud_rts_stack_v1` ([#95](https://github.com/terraform-providers/terraform-provider-opentelekomcloud/issues/95))
```
Comment (niuzhenguo) [inline CHANGELOG.md]: Just want to let you know that you can just use [GH-95] here like ab9b0fd as tf-release-bot will help to translate it 7e15fd3a

**Pull request merged**